src/main/java/appeng/util/item/AEFluidStack.java

```diff
@@ -133,6 +136,10 @@ public static IAEFluidStack loadFluidStackFromPacket(final ByteBuf data) throws
         final byte countReqMadeType = (byte) ((mask2 & 0x3));
         final long countRequestableCrafts = getPacketValue(countReqMadeType, data);
 
+        final byte mask3 = data.readByte();
+        final byte usedPercentType = (byte) ((mask3 & 0x3));
+        final long longUsedPercent = getPacketValue(usedPercentType, data);
```

Comment:
mask2 is still not used up fully
```suggestion
        final byte countReqMadeType = (byte) ((mask2 & 0x3));
        final byte usedPercentType = (byte) ((mask2 & 0xC) >> 2);
        final long countRequestableCrafts = getPacketValue(countReqMadeType, data);
        final long longUsedPercent = getPacketValue(usedPercentType, data);
```